### ISSUE TITLE
Fix the config.json issue

### ISSUE THREAD

**Issue by haooken** [adammhaile/PixelWeb#6]
-Old file in repo
-Should create PixelWeb dir in user home.
-Default Anims directory so that it won't crash on startup(?)

**Comment by adammhaile** [comment]
Been trying to figure out how to do a default dir... it would kinda be
BiblioPixelAnimations, but that would have to be installed first... any
thoughts?

**Comment by haooken** [comment]
Maybe the auto-created pixelweb dir in user home? We'd have to recommend that they pull the BPA repo. I was thinking for this just something so that it doesn't crash on boot.

**Comment by adammhaile** [comment]
Either that or ship with demo anims? Fixing the crash is easy, but we need
to point them to the fact that they need to install other anims.

**Comment by haooken** [comment]
Shipping with a few simple anims would be good. Could you default it to the PixelWeb install directory?

**Comment by adammhaile** [comment]
Maybe... user dir would be better. I'll look into it.

**Comment by haooken** [comment]
Noticed odd behavior when filling in the anim dirs. If I pasted in:
C:\Users\Dan\BTSync\BPGIT\BiblioPixelAnimations\matrix
It would auto-correct some of the slashes starting at the beginning. I was able to save it with this entered in the box:
C:/Users/Dan\BTSync/BPGIT\BiblioPixelAnimations\matrix
When restarting, the server would start, and then if I went back to that box, if I moved the cursor around with the arrow keys or hit HOME and END, it would auto-correct more of the slashes. So in short, it works, but it just looks odd.
